documentation/IDTA-01001/modules/ROOT/pages/spec-metamodel/overview.adoc

```diff
@@ -59,7 +59,7 @@ Data specification templates like the template for IEC 61360 property definition
 
 Besides submodel elements including properties and concept descriptions, other identifiable elements may also use additional templates (xref:spec-metamodel/common.adoc#has-data-specification-attributes[HasDataSpecification]). Data specification templates are selected at design time. 
 
-Figure <<image-metamodel-package-overview>> gives a complete overview of all elements defined in the metamodel and specified in Clause xref:spec-metamodel/nav-spec.adoc[Specification]. The UML packages reflect the structure of Clause xref:spec-metamodel/designators.adoc[Metamodel Specification Details: Designators]. The elements of package "Core" are specified as first class citizens in Clause xref:spec-metamodel/core.adoc[Core Classes], except for their imported packages: the elements of package "SubmodelElements" are specified in Clause xref:spec-metamodel/submodel-elements.adoc[Overview of Submodel Element Types]. Elements of package "Common" are specified in Clause xref:spec-metamodel/common.adoc[Common Attributes]. The elements of package "Reference" are specified in Clause xref:spec-metamodel/referencing.adoc[Referencing in Asset Administration Shells]. Elements from package "Types" are specified in Clause xref:spec-metamodel/datatypes.adoc[Primitive and Simple Data Types]. The only package that is not listed is "Data Specifications (Templates)" because data specifications are handled differently. Data specification templates are explained in Clause ref:data-specifications.adoc[Data Specification Templates].
+ <<image-metamodel-package-overview>> gives a complete overview of all elements defined in the metamodel and specified in Clause xref:spec-metamodel/nav-spec.adoc[Specification]. The UML packages reflect the structure of Clause xref:spec-metamodel/designators.adoc[Metamodel Specification Details: Designators]. The elements of package "Core" are specified as first class citizens in Clause xref:spec-metamodel/core.adoc[Core Classes], except for their imported packages: the elements of package "SubmodelElements" are specified in Clause xref:spec-metamodel/submodel-elements.adoc[Overview of Submodel Element Types]. Elements of package "Common" are specified in Clause xref:spec-metamodel/common.adoc[Common Attributes]. The elements of package "Reference" are specified in Clause xref:spec-metamodel/referencing.adoc[Referencing in Asset Administration Shells]. Elements from package "Types" are specified in Clause xref:spec-metamodel/datatypes.adoc[Primitive and Simple Data Types]. The only package that is not listed is "Data Specifications (Templates)" because data specifications are handled differently. Data specification templates are explained in Clause ref:data-specifications.adoc[Data Specification Templates].
```

Comment:
```suggestion
 <<image-metamodel-package-overview>> gives a complete overview of all elements defined in the metamodel and specified in Clause xref:spec-metamodel/nav-spec.adoc[Specification]. The UML packages reflect the structure of Clause xref:spec-metamodel/designators.adoc[Metamodel Specification Details: Designators]. The elements of package "Core" are specified as first class citizens in Clause xref:spec-metamodel/core.adoc[Core Classes], except for their imported packages: the elements of package "SubmodelElements" are specified in Clause xref:spec-metamodel/submodel-elements.adoc[Overview of Submodel Element Types]. Elements of package "Common" are specified in Clause xref:spec-metamodel/common.adoc[Common Attributes]. The elements of package "Reference" are specified in Clause xref:spec-metamodel/referencing.adoc[Referencing in Asset Administration Shells]. 
Elements from package "Types" are specified in Clause xref:spec-metamodel/datatypes.adoc[Primitive and Simple Data Types]. 
The only package that is not listed is "Data Specifications (Templates)" because data specifications are handled differently. 
Data specification templates are explained in Clause ref:data-specifications.adoc[Data Specification Templates].
```